datadog_lambda/wrapper.py

```diff
@@ -35,12 +35,16 @@
 )
 from datadog_lambda.trigger import extract_trigger_tags, extract_http_status_code_tag
 from datadog_lambda.tag_object import tag_object
+from ddtrace.profiling import profiler
 
 logger = logging.getLogger(__name__)
 
 dd_capture_lambda_payload_enabled = (
     os.environ.get("DD_CAPTURE_LAMBDA_PAYLOAD", "false").lower() == "true"
 )
+profiling_env_var = os.environ.get("DD_PROFILING_ENABLED", "false").lower() == "true"
+service_env_var = os.environ.get("DD_SERVICE", "DefaultServiceName")
+env_env_var = os.environ.get("DD_ENV", "dev")
```

Comment:
should we leave that empty if no env has been set? I'm not sure the default value is "dev" ?

Reply:
Ah yeah we can use `None` here.